client/src/components/Modals/SubmitPitch/index.tsx

```diff
@@ -104,7 +147,16 @@ const SubmitPitchModal: FC<SubmitPitchModalProps> = ({
       onClose={() => setIsOpen(false)}
       {...rest}
     >
-      <Modal.Header content="Submit a pitch" />
+      <Modal.Header>
+        <span style={{ paddingRight: '10px' }}>Submit a pitch</span>
```

Comment:
Avoid inline styling where possible

Reply:
Do you have any recommendations on how to avoid this in this case? Only thing I can think of is create an id and style in the css sheet